runtime/doc/news.txt

```diff
@@ -284,6 +284,8 @@ PERFORMANCE
   highlighting.
 • LSP diagnostics and inlay hints are de-duplicated (new requests cancel
   inflight requests). This greatly improves performance with slow LSP servers.
+• 10x speedup for |vim.treesitter.foldexpr()| (when no parser exists for the
```

Comment:
I would omit the `10x` here since the performance gain depends on a lot of factors (the OS, hardware, anti-viruses etc).

Reply:
It doesn't need to be exact but mentioning a rough estimate is better than just "improved performance". If this matches some benchmarks then it's good.

Reply:
I'm personally not a fan of mentioning stats like these without specifying the conditions of the benchmark tests. Without the context from this PR, a `10x speedup` almost sounds like a buzz headline.

Reply:
it is a buzz headline

Reply:
Fair enough. If the goal of the headline is to get people excited, then I'll stop complaining here.